ui-backend/catalog-ui-search/pom.xml

```diff
@@ -158,19 +158,24 @@
         <dependency>
             <groupId>org.geotools</groupId>
             <artifactId>gt-opengis</artifactId>
-            <version>${opengis.version}</version>
```

Comment:
why does this have to change to a hardcoded version?

Reply:
because there's only a single usage of it, the property probably should have been removed but looks like i left it in there.

Reply:
there's more than 1 usage of it though; it seems like we should use the property and revert these changes